airflow-core/src/airflow/ui/src/i18n/locales/pl/dag.json

```
@@ -0,0 +1,109 @@
+{
+  "allRuns": "Wszystkie uruchomienia",
+  "code": {
+    "bundleUrl": "Adres URL grup Dagów",
+    "bundleVersion": "Wersja gupy Dagów:",
+    "noCode": "Nie znaleziono kodu",
+    "parsedAt": "Przeanalizowano o:"
+  },
+  "details": {
+    "fields": {
+      "bundleVersion": "Wersja grupy Dagów",
+      "catchup": "Nadrabianie zaległości",
+      "concurrency": "Współbieżność",
+      "dagId": "Identyfikator Daga",
+      "dagRunTimeout": "Limit czasu uruchomienia Daga",
+      "defaultArgs": "Domyślne argumenty",
+      "description": "Opis",
+      "endDate": "Data zakończenia",
+      "fileLocation": "Lokalizacja pliku",
+      "hasTaskConcurrencyLimits": "Są ograniczenia współbieżności zadań",
+      "lastExpired": "Ostatnio wygasły",
+      "lastParsed": "Ostatnia analiza",
+      "latestDagVersion": "Najnowsza wersja Daga",
+      "maxActiveRuns": "Maksymalna liczba aktywnych uruchomień",
+      "maxActiveTasks": "Maksymalna liczba aktywnych zadań",
+      "maxConsecutiveFailedDagRuns": "Maksymalna liczba kolejnych nieudanych uruchomień Daga",
+      "params": "Parametry",
+      "startDate": "Data rozpoczęcia",
+      "timezone": "Strefa czasowa"
+    }
+  },
+  "grid": {
+    "buttons": {
+      "resetToLatest": "Przywróć do najnowszego",
+      "toggleGroup": "Przełącz grupę"
+    }
+  },
+  "header": {
+    "buttons": {
+      "dagDocs": "Dokumentacja Daga"
+    },
+    "modals": {
+      "docTitle": "Dokumentacja Daga"
+    },
+    "stats": {
+      "latestDagVersion": "Najnowsza wersja Daga",
+      "latestRun": "Ostatnie uruchomienie",
+      "nextRun": "Następne uruchomienie",
+      "owner": "Właściciel",
+      "schedule": "Harmonogram",
+      "tags": "Etykiety"
+    }
+  },
+  "overview": {
+    "buttons": {
+      "failedRun_one": "Nieudane uruchomienie",
+      "failedRun_other": "Nieudane uruchomienia",
+      "failedTask_one": "Nieudane zadanie",
+      "failedTask_other": "Nieudane zadania"
+    },
+    "charts": {
+      "assetEvent": "Utworzono zdarzenie zasobu"
+    },
+    "failedLogs": {
+      "title": "Ostatnie logi nieudanych zadań",
+      "viewFullLogs": "Zobacz pełne logi"
+    }
+  },
+  "panel": {
+    "buttons": {
+      "options": "Opcje",
+      "showGraph": "Pokaż graf",
+      "showGrid": "Pokaż siatkę"
+    },
+    "dagRuns": {
+      "label": "Liczba uruchomień Daga"
+    },
+    "dependencies": {
+      "label": "Zależności",
+      "options": {
+        "allDagDependencies": "Wszystkie zależności Daga",
+        "externalConditions": "Warunki zewnętrzne",
+        "onlyTasks": "Tylko zadania"
+      },
+      "placeholder": "Zależności"
+    },
+    "graphDirection": {
+      "label": "Kierunek grafu"
+    }
+  },
+  "tabs": {
+    "backfills": "Uzupełnienia wsteczne",
```

```suggestion
    "backfills": "Wypełnienia wsteczne",
```
In the `common.json` we refer to it as `Wypełnienia`